examples/text-generation/run_generation.py

```diff
@@ -317,7 +319,10 @@ def generate(size=None, reduce_recompile=False):
                 profiling_steps=args.profiling_steps,
                 profiling_warmup_steps=args.profiling_warmup_steps,
             ).cpu()
-            return tokenizer.batch_decode(outputs, skip_special_tokens=True)
+            x = tokenizer.batch_decode(outputs, skip_special_tokens=True)
```

Comment:
Lets rename outputs to output_tokens and x to outputs for better convention and understandibility.